editor/import/resource_importer_texture.cpp

```diff
@@ -586,6 +586,18 @@ Error ResourceImporterTexture::import(const String &p_source_file, const String
 				}
 			}
 		}
+
+		if (pack_channels == 3) { // BGRA mode, swapping B and R channels
```

Comment:
```suggestion
		if (pack_channels == 3) { // BGRA mode, swapping B and R channels.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).